exchange/bitswap/decision/peer_request_queue.go

```diff
@@ -156,7 +156,7 @@ func (t *peerRequestTask) SetIndex(i int) {
 
 // taskKey returns a key that uniquely identifies a task.
 func taskKey(p peer.ID, k u.Key) string {
-	return string(p.String() + k.String())
+	return string(p) + string(k)
```

Comment:
yes. lets convert every little thing we do to base58 for no good reason... i'm not the smartest person sometimes.